cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -418,17 +427,15 @@ def __init__(self, *component_gates: cirq.Gate) -> None:
             else:
                 self.component_gates += (gate,)
 
-    def qubit_index_to_gate_and_index(  # pylint: disable=missing-function-docstring
-        self, index: int
-    ) -> Tuple[cirq.Gate, int]:
+    def _qubit_index_to_gate_and_index(self, index: int) -> Tuple[cirq.Gate, int]:
```

Comment:
Does this not need a docstring because it's private?

Reply:
Exactly